scripts/run-pr.sh

```diff
@@ -13,4 +13,4 @@ git rebase master
 npm install
 NODE_ENV=production npm run build
 npm test || true
-npm start
+npm start -- "${@:2}"
```

Comment:
This is bash specific, not sh. We could tell the file to use bash, or use "shift", which seems more common to me.